src/status_im/contexts/settings/wallet/saved_addresses/save_address/view.cljs

```diff
@@ -7,39 +7,72 @@
-                                                             {:on-success
-                                                              [:wallet/add-saved-address-success
-                                                               (i18n/label :t/address-saved)]
-                                                              :on-error
-                                                              [:wallet/add-saved-address-failed]
-                                                              :name address-label
-                                                              :address address-without-prefix
-                                                              :customization-color address-color}])))
-                                           [address address-label address-color])]
+        [selected-networks set-selected-networks]
+        (rn/use-state (network-utils/network-preference-prefix->network-names network-prefixes))
+        chain-short-names (rn/use-memo
+                           #(network-utils/network-names->network-preference-prefix
+                             selected-networks)
+                           [selected-networks])
+        placeholder (i18n/label :t/address-name)
+        open-network-preferences (rn/use-callback
+                                  (fn []
+                                    (rf/dispatch
+                                     [:show-bottom-sheet
+                                      {:theme   :dark
+                                       :shell?  true
+                                       :content (network-preferences-sheet
+                                                 {:address address-without-prefix
+                                                  :color address-color
+                                                  :selected-networks selected-networks
+                                                  :set-selected-networks
+                                                  set-selected-networks})}]))
+                                  [address selected-networks address-color])
+        on-press-save (rn/use-callback
+                       (fn []
+                         (rf/dispatch [:wallet/save-address
+                                       {:on-success
+                                        [:wallet/add-saved-address-success
+                                         (i18n/label :t/address-saved)]
+                                        :on-error
+                                        [:wallet/add-saved-address-failed]
+                                        :name address-label
+                                        :address address-without-prefix
+                                        :customization-color address-color
+                                        :chain-short-names chain-short-names}]))
+                       [address selected-networks address-label
+                        address-color])]
```

Comment:
Nice work, why not? 
```suggestion
                       [address-without-prefix chain-short-names address-label
                        address-color])]
```

Reply:
Ah, we can. I added the `address` as if there is any change, it will affect the `address-without-prefix` as well.
Same for `selected-networks`. I will update it 👍